joint_limits:
  arm_roll_joint:
    has_position_limits: true
    min_position: -3.14
    max_position: 3.14
    has_velocity_limits: true
    max_velocity: 5.969211435
    has_acceleration_limits: true
    max_acceleration: 4.0
  arm_flex_joint:
    has_position_limits: true
    min_position: -1.57
    max_position: 1.57
    has_velocity_limits: true
    max_velocity: 5.969211435
    has_acceleration_limits: true
    max_acceleration: 4.0
  elbow_flex_joint:
    has_position_limits: true
    min_position: -1.57
    max_position: 1.42
    has_velocity_limits: true
    max_velocity: 5.969211435
    has_acceleration_limits: true
    max_acceleration: 4.0
  wrist_flex_joint:
    has_position_limits: true
    min_position: -1.57
    max_position: 1.57
    has_velocity_limits: true
    max_velocity: 5.969211435
    has_acceleration_limits: true
    max_acceleration: 4.0
  hand_motor_joint:
    has_position_limits: true
    min_position: 0
    max_position: 1.57
    has_velocity_limits: true
    max_velocity: 5.969211435
    has_acceleration_limits: true
    max_acceleration: 4.0
  xtion_tilt_joint:
    has_position_limits: true
    min_position: -1.57
    max_position: 1.57
    has_velocity_limits: true
    max_velocity: 5.969211435
    has_acceleration_limits: true
    max_acceleration: 4.0
  xtion_pan_joint:
    has_position_limits: true
    min_position: -1.57
    max_position: 1.57
    has_velocity_limits: true
    max_velocity: 5.969211435
    has_acceleration_limits: true
    max_acceleration: 4.0